logrotate/CMakeLists.txt

```diff
@@ -142,7 +143,7 @@ ENDIF()
 
 SET(${LIBRARY_TO_BUILD}_VERSION_STRING ${VERSION})
 SET_TARGET_PROPERTIES(${LIBRARY_TO_BUILD} PROPERTIES LINKER_LANGUAGE CXX)
-TARGET_LINK_LIBRARIES(${LIBRARY_TO_BUILD}  ${Boost_LIBRARIES} ${ZLIB_LIBRARIES} ${G3LOG})
+TARGET_LINK_LIBRARIES(${LIBRARY_TO_BUILD}  ${Boost_LIBRARIES} ${ZLIB_LIBRARIES} ${G3LOG_LIBRARY})
```

Comment:
Finally got around to review this. 
To make this cohesive can you updated 
```
find_library(g3logger g3logger PATHS /usr/local/probe/lib )
 include_directories(${CMAKE_PREFIX_PATH}/include)
```
 to be 
```
find_library(G3LOG_LIBRARY g3logger PATHS /usr/local/probe/lib )
include_directories(${CMAKE_PREFIX_PATH}/include)
```

And line 14 can be deleted
```
SET(G3LOG_LIBRARY g3logger)
```

Reply:
updated :)